migtests/scripts/oracle/create_oracle_user

```diff
@@ -1,2 +1,9 @@
 #!/usr/bin/env bash
+source migtests/scripts/oracle/env.sh
```

Comment:
Just add a comment at the top stating the purpose of the script i.e. to enable jenkins pipeline to create an oracle user. And relevant links to the code will also help.